tests/GraphQL/Traits/ElementIdentificationTraitTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\DataHubBundle\Tests\GraphQL\Traits;
+
+use PHPUnit\Framework\TestCase;
+use \Pimcore\Bundle\DataHubBundle\GraphQL\Traits\ElementIdentificationTrait;
+
+use Pimcore\Bundle\DataHubBundle\GraphQL\Exception\ClientSafeException;
+
+class TestTrait
+{
+    use ElementIdentificationTrait;
+
+    const BY_ID = "ById";
+    const BY_PATH = "ByPath";
+
+    protected function getElementById($elementType)
+    {
+        return $elementType . self::BY_ID;
+    }
+
+    protected function getElementByPath($elementType)
+    {
+        return $elementType . self::BY_PATH;
+    }
+}
+
+class ElementIdentificationTraitTest extends TestCase
+{
+    const TRAIT_TO_TEST = '\Pimcore\Bundle\DataHubBundle\GraphQL\Traits\ElementIdentificationTrait';
+    const TEST_TYPE = 'object';
+
+    public function testThrowingClientSafeExceptionIfTypeIsMissing()
+    {
+        // Arrange
```

Comment:
I am experiencing the following problem. Phpunit version is 7.5.20

1) ElementIdentificationTraitTest: Throwing client safe exception if type is missing
 Test  ../data-hub/tests/GraphQL/Traits/ElementIdentificationTraitTest.php:testThrowingClientSafeExceptionIfTypeIsMissing

  [Error] Call to undefined method Pimcore\Bundle\DataHubBundle\Tests\GraphQL\Traits\ElementIdentificationTraitTest::expectExceptionMessageMatches()

/home/pimcore-5/demo-basic/dev/pimcore/data-hub/tests/GraphQL/Traits/ElementIdentificationTraitTest.php:49
/home/pimcore-5/demo-basic/vendor/phpunit/phpunit/src/Framework/TestCase.php:1154
/home/pimcore-5/demo-basic/vendor/phpunit/phpunit/src/Framework/TestCase.php:842
/home/pimcore-5/demo-basic/vendor/phpunit/phpunit/src/Framework/TestResult.php:693

Reply:
Ah, finally I found it: the function expectExceptionMessageMatches() was initially named differently (renamed in phpunit 8.4): https://github.com/sebastianbergmann/phpunit/commit/d1199cb2e43a934b51521656be9748f63febe18e
For backward compatible (to phpunit 7.5 / Pimcore 5.7) I renamed the function back to expectExceptionMessageRegExp().